templates/Symbiote/GridFieldExtensions/GridFieldOrderableRowsDragHandle.ss

```diff
@@ -1,2 +1,2 @@
-<span class="handle"><span class="icon"></span></span>
+<div class="handle"><i class="icon font-icon-sort"></i></div>
```

Comment:
Icon class can be changed to font-icon-drag-handle once https://github.com/silverstripe/silverstripe-admin/pull/219 is merged 🙂